core/src/main/java/org/bitcoinj/params/FloMainNetParams.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2013 Google Inc.
+ * Copyright 2015 Andreas Schildbach
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bitcoinj.params;
+
+import org.bitcoinj.core.*;
+import org.bitcoinj.net.discovery.*;
+
+import java.net.*;
+
+import static com.google.common.base.Preconditions.*;
+
+/**
+ * Parameters for the main production network on which people trade goods and services.
+ *
+ * Copied from MainNetParams; see FLO commentary for modifications.
+ */
+public class FloMainNetParams extends AbstractBitcoinNetParams {
+    public static final int MAINNET_MAJORITY_WINDOW = 1000;
+    public static final int MAINNET_MAJORITY_REJECT_BLOCK_OUTDATED = 950;
+    public static final int MAINNET_MAJORITY_ENFORCE_BLOCK_UPGRADE = 750;
+
+    public FloMainNetParams() {
+        super();
+
+        /**
+           Begin section modified for FLO, believed to be correct.
```

Comment:
'believed to be correct'? 

were these extracted from flo source?
are these values good for current FLO? Or 0.15?

Reply:
OK, I make some changes to the commentary.

Those values should be good for all versions of Florincoin.